docs/no_numerical_exp_report.tex

```diff
@@ -56,12 +61,11 @@ \section{Introduction}
 agent, deploys \texttt{Facilities} to meet the front-end and back-end demands of the 
 fuel cycle.
 
-This report describes numerical tests for non-optimizing, deterministic-optimizing and stochastic-optimizing prediction algorithms.
+This report describes numerical tests for the non-optimizing algorithm, namely \gls{ARMA} and \gls{ARCH}.
```

Comment:
should it be numerical experiments (we probably want to be more consistent with what we call the tests/numerical experiments). "This report describes numerical experiments to test the non-optimizing algorithm ... "